charts/nexus-iq/tests/deployment_test.yaml

```diff
@@ -102,6 +104,80 @@ tests:
           path: spec.template.spec.imagePullSecrets
           value:
             - name: top-secret
+  - it: should add an initContainer for fixOwner with defaults
+    set:
+      fixOwner:
+        enabled: true
+    asserts:
+      - equal:
+          path: spec.template.spec.initContainers[0].name
+          value: chown-vols
+      - equal:
+          path: spec.template.spec.initContainers[0].image
+          value: busybox:latest
+      - equal:
+          path: spec.template.spec.initContainers[0].imagePullPolicy
+          value: IfNotPresent
+      - equal:
+          path: spec.template.spec.initContainers[0].securityContext
+          value:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+      - equal:
+          path: spec.template.spec.initContainers[0].command
+          value:
+            - sh
+            - -c
+            - chown -R 1000:1000 /var/log/nexus-iq-server /sonatype-work
+      - equal:
+          path: spec.template.spec.initContainers[0].volumeMounts
+          value:
+            - mountPath: /sonatype-work
+              name: nxiq-pv-data
+            - mountPath: /var/log/nexus-iq-server
+              name: nxiq-pv-log
+            - mountPath: /etc/nexus-iq-server
+              name: config-volume
+  - it: should add an initContainer for fixOwner with all parameters
+    set:
+      fixOwner:
+        enabled: true
+        updateUser: MY-USER
+        updateGroup: MY-GROUP
+      image:
+        pullPolicy: MY-POLICY
+      configYaml:
+        sonatypeWork: /MY/WORK/DIR
```

Comment:
here i set all the values i can for that initContainer and watch them show up in the results as I expect.